dotnet-interactive/JupyterKernelSpecModule.cs

```diff
@@ -20,10 +20,27 @@ public Task<CommandLineResult> InstallKernel(DirectoryInfo sourceDirectory)
             return ExecuteCommand($@"install ""{sourceDirectory.FullName}""", "--user");
         }
 
-        public Task<CommandLineResult> UninstallKernel(DirectoryInfo sourceDirectory)
+        public  DirectoryInfo GetDefaultKernelSpecDirectory()
         {
-            return ExecuteCommand($@"uninstall ""{sourceDirectory.FullName}""");
-        }
+            DirectoryInfo directory;
+            switch (Environment.OSVersion.Platform)
+            {
+                case PlatformID.Win32S:
+                case PlatformID.Win32Windows:
+                case PlatformID.Win32NT:
+                    directory = new DirectoryInfo(Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData), "jupyter", "kernels"));
+                    break;
+                case PlatformID.Unix:
+                    directory = new DirectoryInfo("~/.local/share/jupyter/kernels");
+                    break;
+                case PlatformID.MacOSX:
+                    directory = new DirectoryInfo("~/Library/Jupyter/kernels");
+                    break;
+                default:
+                    throw new PlatformNotSupportedException();
```

Comment:
Thanks for addressing this!
The changes look good to me, but I will leave it to Jon to officially sign off.